testfiles/1ZoneDataCenterCRAC_wPumpedDXCoolingCoil.idf

```diff
@@ -846,6 +846,7 @@
   ElectricEquipment:ITE:AirCooled,
     Data Center Servers,     !- Name
     Main Zone,               !- Zone Name
+	,
```

Comment:
No tabs.  Fill in with full field name and value, not blank here.  Same for 2ZoneDataCenterHVAC_wEconomizer.idf

Reply:
I added one testfile and modified another.